### PR TITLE
styling-PR

### PR DESCRIPTION
**Pull request by gilberthappi**
Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/App.js
Comment (joskalenda) [inline src/App.js:1]: - [ ] Your design is almost perfect, you have done it at 98%, but see, we are expecting from the student more than that, you are being presented with both image
- ORIGINAL TEMPLATE
I am sure there is no additional color in the background or your green tests as you mentioned


- [ ] Your design

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @gilberthappi , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Good commit messages
✅ ... Green linters

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+}
+
+.page {
+  margin: auto;
+  border-radius: 4px;
+  align-items: center;
+}
+
+.wrapper {
+  width: 100vw;
```
Comment (ridwanediallo) [inline src/style.css:58]: - [ **OPTIONAL**] I think your app will look nicer if you remove this fixed width and ensure that the app is centred on the screen as on the template.

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @gilberthappi , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested changes. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**